README.rst

```diff
@@ -438,6 +443,8 @@ Visual Studio Code
 
 You can pass additional arguments to ``darker`` in the ``blackArgs`` option
 (e.g. ``["--diff", "--isort"]``), but make sure at least ``--diff`` is included.
+Also, this won't work if you have any linters enabled on the command line
+or in ``pyproject.toml``.
```

Comment:
that's not true though, it seems that if you don't specify `diff`/`check` in the `pyproject.toml` file - you can override its settings in the cli. (If you do specify them, it doesn't allow to override its configuration).

Though I haven't checked why

Reply:
Yes, including a binary option (like `--diff`, `--check` or `--isort`) in
- just `pyproject.toml`
- just on the command line
- both in `pyproject.toml` and the command line

will enable that option. Is that what you meant?

Also, once enabled in either place, it's currently not possible to override and disable it in the other.

What I mean with "won't work if linters enabled" is that VSCode will not be able to apply the reformattings if there are any linter errors/warnings mixed in Darker's output. Is there a better way to express that?

Also, it seems to me that `--diff --quiet` are hard-coded as options for Black in VSCode, and anything added to `blackArgs` will be appended to those. So that we could fix here as well.

Reply:
I clarified this whole paragraph. Take a look at the newest version, is it now correct and more understandable?

Reply:
Great clarification, thanks.
Is there any reason that boolean-arguments are non overridable? Seems like we should open an issue on that

Reply:
@mayk0gan I agree, boolean options should be overridable. We could also switch from [argparse](https://docs.python.org/3/library/argparse.html) to [click](https://click.palletsprojects.com/) (see #47) to make that easier.